cmd/container-freezer/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"os"
+
+	"go.uber.org/zap"
+	pkglogging "knative.dev/pkg/logging"
+)
+
+func main() {
+	logger, _ := pkglogging.NewLogger("", "")
+	defer flush(logger)
```

Comment:
I don't think this ever gets called, since nothing ever causes http.ListenAndServe to return. We'd need to add a call to `server.Shutdown` on TERM signals for this line to make sense, I think.

Reply:
Maybe it's simpler to just take it out? I used [queue proxy](https://github.com/knative/serving/blob/main/cmd/queue/main.go#L138-L140) as an example of how to set this up, but if it's not needed, then may as well remove it.